test/framework/filetools.py

```diff
@@ -1009,7 +1009,7 @@ def test_back_up_file(self):
         new_file = [x for x in test_files if x not in known_files][0]
         self.assertTrue(new_file.startswith('test.txt.bak_'))
         known_files = os.listdir(os.path.dirname(fp))
-        self.assertTrue(ft.read_file(first_normal_backup), txt)
+        self.assertEqual(ft.read_file(first_normal_backup), txt)
```

Comment:
these change the behaviour, but that looks intended

Reply:
Yes, this was a bug before.